ca/server.go

```diff
@@ -402,6 +340,13 @@ func (s *Server) Run(ctx context.Context) error {
 		state.EventUpdateNode{},
 		state.EventUpdateCluster{},
 	)
+
+	// Do this after updateCluster has been called, so isRunning never
```

Comment:
When an RPC is invoked, it first checks `isRunning`. `isRunning` checks for `s.ctx` to determine whether RPCs can run yet. So it's important to set `s.ctx` only after we initialize the relevant variables.